src/main/java/ChinChin/task/Event.java

```diff
@@ -0,0 +1,59 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents an event task with starting and ending date
+ */
+public class Event extends Task {
+    private LocalDateTime starting;
+    private LocalDateTime ending;
```

Comment:
```suggestion
    private LocalDateTime startingDate;
    private LocalDateTime endingDate;
```

You may consider adding a Date to make the variable name easier to understand:)